experimental/PropertyDDS/packages/property-changeset/src/test/array.spec.ts

```diff
@@ -673,14 +673,14 @@ describe("Array Operations", function () {
 							"array<Float64>",
 						);
 
-						let initialInserts = [];
-						let finalInserts = [];
-						if (additionalInserts === " with insert at the beginning") {
-							initialInserts = [[0, generateNamedEntities(2, undefined, "number")]];
-						}
-						if (additionalInserts === " with insert at the end") {
-							finalInserts = [[10, generateNamedEntities(2, undefined, "number")]];
-						}
+						const initialInserts =
+							additionalInserts === " with insert at the beginning"
+								? [[0, generateNamedEntities(2, undefined, "number")]]
+								: [];
+						const finalInserts =
+							additionalInserts === " with insert at the end"
+								? [[10, generateNamedEntities(2, undefined, "number")]]
+								: [];
```

Comment:
Ternaries introduced to avoid unnecessary reassignment (so variables could be const).